discord/threads.py

```diff
@@ -121,6 +121,11 @@ class Thread(Messageable, Hashable):
         This is always ``True`` for public threads.
     archiver_id: Optional[:class:`int`]
         The user's ID that archived this thread.
+
+        .. note::
+            This field has been removed from the :ddocs:`Thread Metadata Object <resources/channel#thread-metadata-object>`.
+            The ``archiver_id``, therefore, will always be ``None`` and can only be obtained via the audit log now.
```

Comment:
```suggestion
        .. note::
            Due to an API change, the ``archiver_id``, therefore, will always be ``None`` and can only be obtained via the audit log now.
```

Simplify the wording.